Jenkinsfile_k8s

```diff
@@ -1,3 +1,5 @@
+@Library('pipeline-library@pull/822/head') _
+
```

Comment:
```suggestion
```

Added to test https://github.com/jenkins-infra/pipeline-library/pull/822 (with success), not needed anymore.